Cargo.toml

```diff
@@ -28,7 +28,7 @@ resolver = "2"
 exclude = ["node-graph/gpu-compiler"]
 
 [workspace.dependencies]
-specta = { git = "https://github.com/0HyperCube/specta.git", rev = "d28cd72ea4ea2f6eaa85e0622d3a98340dbb43d3", features = [
+specta = { git = "https://github.com/0HyperCube/specta.git", rev = "c47a22b4c0863d27bc47529f300de3969480c66d", features = [
```

Comment:
Just checking, does the reason you had to maintain your own fork still persist or would we be able to switch back to their official repo now? @0HyperCube

Reply:
Two reasons @Keavon 
- They specify glam=0.22 whereas we use glam=0.24 so the encoding doesn't work.
- Their current release doesn't allow doc comments and produces a compile error.
```
error[E0659]: `doc` is ambiguous
  --> editor/src/messages/layout/utility_types/misc.rs:27:2
   |
27 |     /// The color swatch for the working colors and a flip and reset button found at the bottom of the tool shelf.
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ambiguous name
   |
   = note: ambiguous because of a name conflict with a builtin attribute
   = note: `doc` could refer to a built-in attribute
note: `doc` could also refer to the derive helper attribute defined here
  --> editor/src/messages/layout/utility_types/misc.rs:4:83
   |
4  | #[derive(PartialEq, Clone, Debug, Hash, Eq, Copy, Serialize, Deserialize, specta::Type)]
   |                                                                                   ^^^^
```

Reply:
Gotcha, thanks. Would you mind including that in a comment?